ScriptHookVDotNet.sln

```diff
@@ -1,8 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2013 or higher
```

Comment:
Guess that's what I get for assuming it was just a comment, lol. I'll fix it tomorrow when I get a chance. Ideally, what should it be? 2013 or 14?